PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -178,16 +182,34 @@ public void GetPersonByIdExceptionThrow()
         public async Task UpdatePersonSuccessfulUpdates()
```

Comment:
Please additional test coverage - for example, a test to check what happens if the person doesn't exist and a test checking the behaviours when exceptions occur.